prow/plugins/plugins.go

```diff
@@ -479,3 +480,14 @@ func (pa *PluginAgent) getPlugins(owner, repo string) []string {
 
 	return plugins
 }
+
+// GetPluginCounter returns premethus counter for a given plugin
+func GetPluginCounter(m *prometheus.CounterVec, pluginName string) prometheus.Counter {
```

Comment:
Currently the caller of this func could specify any `CounterVec` for `m` even though this function only makes sense for a specific `CounterVec`, `pluginsCounter`.
If you make this a method of `*hook.Metrics` then the caller won't have to specify `m`.